pkgs/tools/compression/zstd/playtests-darwin.patch

```diff
@@ -1,18 +1,28 @@
 --- a/tests/playTests.sh
 +++ b/tests/playTests.sh
-@@ -109,5 +109,2 @@ esac
+@@ -112,17 +112,10 @@ case "$OS" in
+ esac
+ 
  case "$UNAME" in
 -  Darwin) MD5SUM="md5 -r" ;;
 -  FreeBSD) MD5SUM="gmd5sum" ;;
+-  NetBSD) MD5SUM="md5 -n" ;;
```

Comment:
What's the connection between these changes (adding netbsd to this patch) and the 1.48 -> 1.49 update? Also, what's the purpose of this patch?

Reply:
Presumably this is just adapting the patch to the upstream changes; upstream added support for NetBSD.

This patch appears to remove the specialcasing for checksumming for all platforms, presumably because we want to use the One True md5sum command that we get from the Nix environment rather than whatever the platform-specific-usually-installed-by-default one is.

Reply:
@lukegb: Thanks, makes sense.

@LeSuisse: It might be helpful to explain the [purpose of the patch](https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md) in a comment, since it's non-obvious just looking at the patch file.

Reply:
I confirm I just adapted the existing patch to uptream changes. I will add a comment ;) .